### ISSUE TITLE
How to build this thing?

### ISSUE THREAD

**Issue by dimonik** [mslosarz/nextrtc-signaling-server#1]
Hi Marcin!

Can you explain how to build & run this server?

**Comment by mslosarz** [comment]
Hello

First you have to build nextrtc-signaling-server (by mvn install)
Then you can see an example (where library is used
nextrtc-example-videochat -> simple usage,
nextrtc-videochat-with-rest -> in progress... handling signals with storing then by hibernate)

If you have some compilation problems in yours IDE probably you have to install lombok

Best regards

**Comment by pancitmiki** [comment]
Hi Marcin,

Is lombok installable through maven by adding it as a dependency in the pom
file?

Thank you.

**Comment by mslosarz** [comment]
To build project by maven, lombok dependency is enough to success. 
If you want to use lombok in your IDE you have to download plugin (for IntelliJ) or execute jar provided by lombok project (Eclipse) for more details see https://projectlombok.org/

**Comment by demoontz** [comment]
Hello!
could u tell what is working on nextrtc-videochat-with-rest? can we make call for 2 people?

**Comment by mslosarz** [comment]
I think that it shouldn't be problem, but I can make a call with you
tomorrow, because today I'm at work to 8 p. m. (in UTC time)

In nextrtc-videochat-with-rest I checked it is possible to plug signaling
server to application with hibernate. I created some handlers and when some
event occurs (for example someone has joined) I've tried to store this
information.

I have example (I know that it doesnt work now, but it's problem with proxy
configuration, not with application), this example works fine on tomcat8,
what more in example in view (somewhere in index.html) there is hardcoded
endpoint value try to change it :)



2015-12-18 16:16 GMT+01:00 demoontz <notifications@github.com>:

> Hello!
> could u tell what is working on nextrtc-videochat-with-rest? can we make
> call for 2 people?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mslosarz/nextrtc-signaling-server/issues/1#issuecomment-165801626>
> .
>



-- 
Marcin Ślósarz
